package.json

```
@@ -2,13 +2,19 @@
   "name": "graphql-subscriptions-client",
   "version": "0.13.0",
   "description": "A simpler client for graphql subscriptions based on apollographql/subscriptions-transport-ws",
-  "main": "client.js",
+  "main": "dist/index.js",
```

Could you add a module property pointing to the esm files? For me, the big advantage of this library is that it works with native ES module build tools, which needs to know about these.

```suggestion
  "module": "dist/esm//index.js",
  "main": "dist/index.js",
```